reference/5.1/Microsoft.PowerShell.Utility/Invoke-RestMethod.md

```diff
@@ -111,19 +112,6 @@ $Body = @{
 Invoke-RestMethod -Method Post -Uri $url -Credential $Cred -Body $body -OutFile output.csv
 ```
 
-```
-cmdlet Get-Credential at command pipeline position 1
```

Comment:
I have a feeling this block was supposed to be a set of either parsed results or the contents of the CSV.